src/utils.ts

```diff
@@ -480,19 +480,22 @@ const isLumigoStackTrace = (input) => {
   return LUMIGO_STACK_PATTERNS.some((word) => word.test(input));
 };
 
+export const removeLumigoFromError = (stacktrace: string) => {
+  const stackArr = stacktrace.split('\n');
+
+  const cleanedStackArr = stackArr.filter((v) => !isLumigoStackTrace(v));
+
+  return cleanedStackArr.join('\n');
+};
 export const removeLumigoFromStacktrace = (handleReturnValue) => {
   // Note: this function was copied to the auto-instrument-handler. Keep them both up to date.
```

Comment:
what is the auto-instrument-handler? should I add these changes there as well? @harelmo-lumigo